src/locale/pt-br/texts.po.json

```diff
@@ -0,0 +1 @@
le:","msgstr":["Disponível:"]},"Locked:":{"msgid":"Locked:","msgstr":["Bloqueado:"]},"Transaction history":{"msgid":"Transaction history","msgstr":["Histórico de transações"]},"Mint":{"msgid":"Mint","msgstr":["Criar"]},"Melt":{"msgid":"Melt","msgstr":["Destruir"]},"${ type } output delegated!":{"msgid":"${ type } output delegated!","msgstr":["${ type } output delegado!"]},"mint":{"msgid":"mint","msgstr":["criar"]},"melt":{"msgid":"melt","msgstr":["destruir"]},"Address of the new ${ type } authority":{"msgid":"Address of the new ${ type } authority","msgstr":["Endereço da nova autoridade de ${ type }"]},"Create another ${ type } output for you?":{"msgid":"Create another ${ type } output for you?","msgstr":["Criar outro ${ type } output para você?"]},"Leave it checked unless you know what you are doing":{"msgid":"Leave it checked unless you know what you are doing","msgstr":["Deixe marcado a não ser que você saiba o que está fazendo"]},"${ label } outputs destroyed!":{"msgid":"${ label } outputs destroyed!","msgstr":["${ label } outputs destruídos!"]},"Are you sure you want to destroy <strong>${ quantity } ${ type }</strong> authority ${ plural }?":{"msgid":"Are you sure you want to destroy <strong>${ quantity } ${ type }</strong> authority ${ plural }?","msgstr":["Você tem certeza que deseja destruir <strong>${ quantity } ${ type }</strong> authority ${ plural }?"]},"Destroy":{"msgid":"Destroy","msgstr":["Destruir"]},"Can't find outputs to melt the amount requested.":{"msgid":"Can't find outputs to melt the amount requested.","msgstr":["Não foi possível encontrar outputs para destruir a quantidade desejada."]},"${ prettyAmountValue } ${ _this.props.token.symbol } melted!":{"msgid":"${ prettyAmountValue } ${ _this.props.token.symbol } melted!","msgstr":["${ prettyAmountValue } ${ _this.props.token.symbol } destruídos!"]},"The total amount you have is only ${ prettyWalletAmount }.":{"msgid":"The total amount you have is only ${ prettyWalletAmount }.","msgstr":["A quantidade total que você tem é de apenas ${ prettyWalletAmount }."]},"Create another melt output for you?":{"msgid":"Create another melt output for you?","msgstr":["Criar outro melt output para você?"]},"Melt tokens":{"msgid":"Melt tokens","msgstr":["Destuir tokens"]},"${ prettyAmountValue } ${ _this.props.token.symbol } minted!":{"msgid":"${ prettyAmountValue } ${ _this.props.token.symbol } minted!","msgstr":["${ prettyAmountValue } ${ _this.props.token.symbol } criados!"]},"Address is required when not selected automatically":{"msgid":"Address is required when not selected automatically","msgstr":["Endereço é obrigatório quando não for selecionado automaticamente"]},"Automatically select address to receive new tokens":{"msgid":"Automatically select address to receive new tokens","msgstr":["Automaticamente selecionar endereço para receber novos tokens"]},"Create another mint output for you?":{"msgid":"Create another mint output for you?","msgstr":["Criar outro mint output para você?"]}}}}
```

Comment:
Are we keeping these? I can't recall what we decided for the mobile wallet.

Reply:
Just removed it and added to the gitignore. Can you check again, please?